### PR TITLE
feat: 🎸 https://github.com/vuejs/eslint-plugin-vue/issues/1492

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**
Resolved https://github.com/vuejs/eslint-plugin-vue/issues/1492

### CODE REVIEW

**Comment by IWANABETHATGUY** [comment]
I just run eslint:fix, and make so many changes, what should i do ? @ota-meshi

File: lib/rules/this-in-template.js [hunk 1]
```diff
@@ -116,7 +120,10 @@ module.exports = {
...
                       node: reference.id,
                       loc: reference.id.loc,
-                      message: "Expected 'this'."
+                      message: "Expected 'this'.",
+                      fix(fixer) {
+                        return fixer.insertTextBefore(reference.id, 'this.')
+                      }
                     })
                   }
                 }
```
Comment (IWANABETHATGUY) [inline lib/rules/this-in-template.js:129]: I just chagne this-in-template.js and test/this-in-template, the rest just eslint autofix

**Review by ota-meshi — APPROVED** [review]
LGTM! Thank you!

**Comment by ota-meshi** [comment]
Also thank you for the format. I will merge it.

**Pull request merged**